app/main/posts/views/filters/active-search-filters.html

```diff
@@ -1,3 +1,11 @@
+<div class="form-field saved-search" ng-show="collection">
+    <span class="bug">
```

Comment:
so, IDK about this. I didn't want to go into a refactor that made the saved search logic more confusing , so I just opted for having collections in the scope. Which seems the lesser evil. I think..